generators/server/files.js

```diff
@@ -574,35 +574,31 @@ function writeFiles() {
                     this.template(`${SERVER_MAIN_SRC_DIR}package/domain/User.java.ejs`, `${javaDir}domain/User.java`);
                     this.template(`${SERVER_TEST_SRC_DIR}package/web/rest/AccountResourceIntTest.java.ejs`, `${testDir}web/rest/AccountResourceIntTest.java`);
                     this.template(`${SERVER_TEST_SRC_DIR}package/security/SecurityUtilsUnitTest.java.ejs`, `${testDir}security/SecurityUtilsUnitTest.java`);
+                    this.template(`${SERVER_MAIN_RES_DIR}config/liquibase/users.csv.ejs`, `${SERVER_MAIN_RES_DIR}config/liquibase/users.csv`);
+                    this.template(`${SERVER_MAIN_SRC_DIR}package/domain/Authority.java.ejs`, `${javaDir}domain/Authority.java`);
+                    this.template(`${SERVER_MAIN_SRC_DIR}package/service/UserService.java.ejs`, `${javaDir}service/UserService.java`);
+                    this.template(`${SERVER_MAIN_SRC_DIR}package/service/dto/package-info.java.ejs`, `${javaDir}service/dto/package-info.java`);
```

Comment:
looks like you have a formatting issue here

Reply:
I don't see the issue you're referring to. Can you please be more specific?

Reply:
I see some spaces at the beginning of the line - nothing important